devel/icu/Portfile

```diff
@@ -52,7 +52,7 @@ subport ${name}-bootstrap {
     compiler.whitelist      clang llvm-gcc-4.2 gcc-4.2 apple-gcc-4.2
 }
 # Also needed by later clangs.
-if {${os.platform} eq "darwin" && ${os.major} < 11 && ${cxx_stdlib} eq "libc++"} {
+if {${os.platform} eq "darwin" && ${os.major} < 12 && ${cxx_stdlib} eq "libc++"} {
     clang_dependency.extra_versions 3.7
```

Comment:
Any reason not to just always blacklist 3.7? It's possible to have clang++ < 500 on 10.8 too.

Reply:
There is no particular reason.
For ports that I do no maintain, I try to make the minimal set of changes that will fix the problem.